Makefile

```diff
@@ -313,19 +311,21 @@ ifneq ($(wildcard $(XSMM_DIR)/lib/libxsmm.*),)
 endif
 
 # OCCA Backends
+OCCA_BACKENDS = /cpu/occa /gpu/occa /*/occa
```

Comment:
> What is the semantic of `/*/occa`?

I use the available backends and make a decision on the "fastest" 
```
   CUDA/HIP > OpenCL > OpenMP > Serial
```

> We still have other names, like /omp/occa, documented in a bunch of places.

I like that we use the hardware/platform type as the first part and wanted to keep it consistent. I can update the documentation if we're ok with this naming convention

We can also pass the properties to the device as the last argument, I can also update the docs on this part

Reply:
@jeremylt @YohannDudouit :wave: How does this sound?

Reply:
I think it would be good to put the full list of supported OCCA backends so we make sure everything runs against the test suite.

Perhaps `/gpu/occa/cuda`, `/gpu/occa/hip`, and so on?

I like the option `/*/occa`. I've wanted to include `/cpu/self/*/serial` and `/cpu/self/*/blocked`. I would let me set separate default ceeds for multigrid levels (for elasticity, serial when (P+1)*ncomp > 16 else blocked)

Reply:
I wonder if `/occa` wouldn't be enough instead of `/*/occa`, but if you see other usage for the `*` I'm fine with it.